store/migrations/migration0/migrate.go

```diff
@@ -23,288 +15,32 @@ func (m Migration) Timestamp() string {
 }
 
 func (m Migration) Migrate(orm *orm.ORM) error {
-	orm.InitializeModel(&JobSpec{})
-	orm.InitializeModel(&JobRun{})
-	orm.InitializeModel(&Initiator{})
-	orm.InitializeModel(&Tx{})
-	orm.InitializeModel(&TxAttempt{})
-	orm.InitializeModel(&BridgeType{})
-	orm.InitializeModel(&IndexableBlockNumber{})
-	orm.InitializeModel(&User{})
-	orm.InitializeModel(&Session{})
-	orm.InitializeModel(&ServiceAgreement{})
-	return nil
-}
-
-type Time struct {
-	time.Time
-}
-
-// UnmarshalJSON parses the raw time stored in JSON-encoded
-// data and stores it to the Time field.
-func (t *Time) UnmarshalJSON(b []byte) error {
-	var n json.Number
-	if err := json.Unmarshal(b, &n); err != nil {
-		return err
-	}
-	newTime, err := dateparse.ParseAny(n.String())
-	t.Time = newTime.UTC()
+	// orm.DB.LogMode(true) to trace sql commands
```

Comment:
Do we need this comment anymore?

Also, could DRY the below logic up a bit with `multierr.Combine`.

Reply:
I'll ✂️ the comment and try `multierr.Combine`